app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source.rb

```diff
@@ -4,4 +4,8 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScri
   include ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::TowerApi
 
   FRIENDLY_NAME = "Ansible Automation Inside Project".freeze
+
+  def self.display_name(number = 1)
+    n_('Repository (Embedded Ansible)', 'Repositories (Embedded Ansible)', number)
```

Comment:
This name should be in-sync with the `FRIENDLY_NAME` defined in Line 6, right?

Also, isn't that `Ansible Automation Inside` is the PM approved name? Instead of `Embedded Ansible` ?